### PR TITLE
feat(new section): Add a EOSIO Core Section

### LINKED ISSUE #233: Nodeos , Cleos and EOSIO.CDT section
Add new section in Learn EOSIO about Nodeos, Cleos and EOSIO.CDT

### PR DESCRIPTION
**Pull request by Edgar097**
### GH Issue

Resolves #233   

### Steps to test
1. Go to learn eos
1. Open EOSIO core

#### CheckList
- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

File: docs/eos-learn/eosio-core.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+## Getting Started
+
+As a way to help you start in EOSIO world, here is a guide on how to install the EOSIO Contract Development Toolkit.
+If you are using a different OS rather than windows visit [this site](https://developers.eos.io/welcome/latest/getting-started/development-environment/before-you-begin) to see the prerequisites and skip to [Install Binaries](#install-binaries).
+This installation guide will aim for Windows users, unfortunately, EOSIO does not provide PowerShell ports and instructions for this OS. As a developer you have two options *one* to use a Virtual Machine or, like in this case a Linux Terminal on Windows. Follow these steps:
+
+- Go to your Control Panel
+- Go to Programs -> Programs and Features
+- Then, click **Turn Windows features on or off**
```
Comment (leisterfrancisco) [inline docs/eos-learn/eosio-core.md:51]: > Turn Windows features on or off

Is the before option right? I mean, is not just **on** or just **off**

**Review by leisterfrancisco — COMMENTED** [review]
Good work 👍

**Review by leisterfrancisco — APPROVED** [review]
Good work 👍

**Review by steph222 — APPROVED** [review]
😄 Good Job!

**Pull request merged**